arch/risc-v/src/common/riscv_exception_common.S

```diff
@@ -83,20 +83,6 @@ exception_common:
   addi       sp, sp, -XCPTCONTEXT_SIZE
   save_ctx   sp
```

Comment:
Not really a part of this commit, but one option is to rename 
`save_ctx -> riscv_saveintregs`
`load_ctx -> riscv_loadintregs`

Because that is what they do

Reply:
could we define three macro: riscv_savectx, riscv_savecpu and riscv_savefpu?